testprojects/src/scala/coursier_resolve.lockfile

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Let's bootstrap a convention for v2 that we use the `testprojects/src/jvm` root for things. I often wished that we could have done it at Twitter, since it takes the guesswork of "is it Scala, or is it Java?" out of finding the file for a classname.